.github/workflows/scripts/slack_gem_notifications/notifications_methods.rb

```diff
@@ -39,10 +39,22 @@ def gem_updated?(versions)
   Time.now.utc - Time.parse(versions[0]['created_at']) < 24 * 60 * 60
 end
 
+def github_diff(gem_name, newest, previous)
+  diff = HTTParty.get("https://github.com/#{gem_name}/#{gem_name}/compare/v#{previous}...v#{newest}")
+
+  diff.success?
+end
+
 def send_bot(gem_name, versions)
-  abort "Expected exactly 2 version numbers in the 'versions' array" unless versions.size == 2
+  abort("Expected exactly 2 version numbers in the 'versions' array") unless versions.size == 2
   newest, previous = versions[0]['number'], versions[1]['number']
-  HTTParty.post(ENV['SLACK_GEM_NOTIFICATIONS_WEBHOOK'],
-    headers: {'Content-Type' => 'application/json'},
-    body: {text: "A new gem version is out! Gem: #{gem_name}, #{previous} -> #{newest}."}.to_json)
+  if github_diff(gem_name, newest, previous)
+    HTTParty.post(ENV['SLACK_GEM_NOTIFICATIONS_WEBHOOK'],
+      headers: {'Content-Type' => 'application/json'},
+      body: {text: "A new gem version is out :sparkles: <https://rubygems.org/gems/#{gem_name}|*#{gem_name}*>, #{previous} -> #{newest}\n\n<https://github.com/#{gem_name}/#{gem_name}/compare/v#{previous}...v#{newest}|Check out the git diff>"}.to_json)
```

Comment:
Since the interpolation for the GitHub link also occurs in `send_bot`, would you be open to making that its own method so it can be referenced in both spots?

Reply:
(also, I ❤️ the ✨ )

Reply:
Something similar could also be done with the rubygems URL since it is referenced in both conditions.